oio/xcute/common/job.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, conf, logger=None):
         self.conf = conf
         self.logger = logger or get_logger(self.conf)
 
-    def sanitize_config(self, job_config):
+    @classmethod
+    def sanitize_config(cls, job_config):
```

Comment:
Can we change back `get_tasks` to be static as well?

Reply:
In all our code, we give the configuration and the logger in the constructor. So, I prefer to keep this same logic.